misc/docs/source/tutorial/code_snippets/retrieving_data_from_datacenters.rst

```diff
@@ -2,43 +2,123 @@
 Retrieving Data from Data Centers
 =================================
 
-----------------
-ArcLink Protocol
-----------------
-
-In the following example uses the :mod:`obspy.arclink` module in order to
-retrieve waveforms data as well as poles and zeros from a remote server
-via the `ArcLink <http://www.seiscomp3.org/wiki/doc/applications/arclink>`_
-protocol. The retrieved poles and zeros are then used to correct for the
-instrument response and to simulate a 1Hz instrument with damping 0.707.
+This section is intended as a small guide to help you choose the best way to
+download data for a given purpose using ObsPy. Keep in mind that data centers
+and web services are constantly changing so this recommendation might not be
+valid anymore at the time you read this. For actual code examples, please see
+the documentation of the various modules.
 
 .. note::
-    The default client needs to open port 18002 to the host webdc.eu via TCP/IP
-    in order to download the requested data. Please make sure that no firewall
-    is blocking access to this server/port combination.
+
+    The most common use case is likely to download waveforms and event/station
+    meta information. In almost all cases you will want to use the
+    :mod:`obspy.fdsn` module for this. It supports the largest amount of data
+    centers and uses the most modern data formats. There are still a number of
+    reasons to choose a different module but please make sure you have one.
+
+---------------------
+The FDSN Web Services
+---------------------
+
++----------------------+--------------------------------+
+| Available Data Types | Format                         |
++======================+================================+
+| Waveforms            | MiniSEED and optionally others |
++----------------------+--------------------------------+
+| Station Information  | StationXML and Text            |
++----------------------+--------------------------------+
+| Event Information    | QuakeML and Text               |
++----------------------+--------------------------------+
 
 .. note::
-    The ``user`` keyword in the following example is used for identification
-    with the ArcLink server as well as for usage statistics within the data
-    center, so please provide a meaningful user id such as your email address.
 
-.. plot:: tutorial/code_snippets/retrieving_data_from_datacenters_1.py
-   :include-source:
+    Not all data providers offer all three data types. Many offer only one or two.
+
+If you want to requests waveforms, or station/event meta information **you will
+most likely want to use the** :mod:`obspy.fdsn` **module**. It is able to
+request data from any data center implementing the `FDSN web services
+<http://www.fdsn.org/webservices/>`_. Data from a large number of centers like
```

Comment:
s/like/such as/
s/large/great/ or s/large/high/ but the sentence still seems of either way.